packages/docs/content/migration-guides/normalized-theme-colors.mdx

```diff
@@ -0,0 +1,441 @@
+
+<ColorMigrationTable
+  colorRows={[
+    [{ name: 'secondary-lightest', value: '#ecf4fa' }, { name: 'secondary-lightest',  value: '#e7f3e7' }],
+    [{ name: 'secondary-lighter',  value: '#c5dff1' }, { name: 'secondary-lighter',   value: '#89c487' }],
+    [{ name: 'secondary-light',    value: '#78b4dd' }, { name: 'secondary-light',     value: '#2a9526' }],
+    [{ name: 'secondary',          value: '#3e94cf' }, { name: 'secondary',           value: '#12890e' }],
+    [{ name: 'secondary-dark',     value: '#3885ba' }, { name: 'secondary-dark',      value: '#107b0d' }],
+    [{ name: 'secondary-darker',   value: '#2b6891' }, { name: 'secondary-darker',    value: '#0d600a' }],
+    [{ name: 'secondary-darkest',  value: '#1f4a68' }, { name: 'secondary-darkest',   value: '#094507' }],
+  ]}
+/>
+
+</ThemeContent>
+<ThemeContent onlyThemes={['medicare']}>
+
+Secondary colors didn't change for the Medicare theme.
+
+<ColorMigrationTable
+  colorRows={[
+    [{ name: 'secondary-lightest', value: '#e9ecf1' }, { name: 'secondary-lightest',  value: '#e9ecf1' }],
+    [{ name: 'secondary-lighter',  value: '#bcc5d4' }, { name: 'secondary-lighter',   value: '#bcc5d4' }],
+    [{ name: 'secondary-light',    value: '#62779b' }, { name: 'secondary-light',     value: '#62779b' }],
+    [{ name: 'secondary',          value: '#1e3c70' }, { name: 'secondary',           value: '#1e3c70' }],
+    [{ name: 'secondary-dark',     value: '#1b3665' }, { name: 'secondary-dark',      value: '#1b3665' }],
+    [{ name: 'secondary-darker',   value: '#152a4e' }, { name: 'secondary-darker',    value: '#152a4e' }],
+    [{ name: 'secondary-darkest',  value: '#0f1e38' }, { name: 'secondary-darkest',   value: '#0f1e38' }],
+  ]}
+/>
+
+</ThemeContent>
+
+
+<ThemeContent onlyThemes={['cmsgov']}>
+
+Secondary colors didn't change for the CMS.gov theme.
+
+<ColorMigrationTable
+  colorRows={[
+    [{ name: 'secondary-lightest', value: '#ecfcf3' }, { name: 'secondary-lightest',  value: '#ecfcf3' }],
+    [{ name: 'secondary-lighter',  value: '#c6f6db' }, { name: 'secondary-lighter',   value: '#c6f6db' }],
+    [{ name: 'secondary-light',    value: '#7bebac' }, { name: 'secondary-light',     value: '#7bebac' }],
+    [{ name: 'secondary',          value: '#42e288' }, { name: 'secondary',           value: '#42e288' }],
+    [{ name: 'secondary-dark',     value: '#3bcb7a' }, { name: 'secondary-dark',      value: '#3bcb7a' }],
+    [{ name: 'secondary-darker',   value: '#2e9e5f' }, { name: 'secondary-darker',    value: '#2e9e5f' }],
+    [{ name: 'secondary-darkest',  value: '#217144' }, { name: 'secondary-darkest',   value: '#217144' }],
+  ]}
+/>
+
+</ThemeContent>
+
+
+## Status colors
+
+An explanation about status colors
```

Comment:
What were we going to say here?

Reply:
What status colors are used for or why we have status colors and not their non-semantic forms